src/navigations/root-stack.js

```diff
@@ -142,6 +142,14 @@ RootNavigator.propTypes = {
 const AuthenticatedStack = createNativeStackNavigator();
 
 const AuthenticatedNavigator = () => {
+  const withKeyboardWrapper = (Component) => {
+    return (props) => (
+      <KeyboardWrapper>
+        <Component {...props} />
+      </KeyboardWrapper>
+    );
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This higher-order function `withKeyboardWrapper` is a good way to wrap each screen with the `KeyboardWrapper` component. This ensures that all screens will have consistent keyboard behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->